contrib/devtools/copyright_header.py

```diff
@@ -313,7 +316,7 @@ def report_cmd(argv):
 # query git for year of last change
 ################################################################################
 
-GIT_LOG_CMD = "git log --pretty=format:%%ai %s"
+GIT_LOG_CMD = "git log --pretty=format:%%ai --no-merges %s"
```

Comment:
Does this change belong here?

Reply:
It's needed to get the correct list of changed files where the copyright is updated in this pull request. But it could be put into a separate pull request. I don't have a strong preference here. What do you think?

Reply:
If I am not mistaken then a lot of these changes are what is called a scripted diff, right?

I think we should try to use this. A scripted diff does not necessarily have to stand alone as a pull request, but it is an isolated commit with scripted changes that can be reviewed _airquote_ simply _airquote_ by reviewing the script.

https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#scripted-diffs

I believe travis even checks scripted diffs and whether the executes script actually reduces the result as advertised in the commit. At least in bitcoin it does.

Reply:
Ah, good point. Upstream doesn't use the scripted diff for this particular use case but I will give it a try.

Reply:
> https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#scripted-diffs

That's quite cool. So, just rebasing this PR and changing the commit's message this would execute this check?

Reply:
> That's quite cool. So, just rebasing this PR and changing the commit's message this would execute this check in Travis?

Yeah. It works. You can see the check running in the lint step on Travis on #1072.